### PR TITLE
Scope implementation

### PR DESCRIPTION
**Pull request by iramiller**
## Description

Adds implementation of Scope object to metadata module

closes: #53 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,8 +34,16 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+* Add support for Scope objects to Metadata module #53
+* Denom Metadata config for nhash in testnet #42
+* DenomMetadata support for marker module #47
```
Comment (channa-figure) [inline CHANGELOG.md:39]: Add a space to DenomMetadata like the line above.

**Review by channa-figure — COMMENTED** [review]
Just a single space issue in the CHANGELOG

**Review by channa-figure — APPROVED** [review]
LGTM

**Review by dwedul-figure — APPROVED** [review]

**Review by mtps — APPROVED** [review]

**Review by dpederson-figure — APPROVED** [review]
Rubber stamp

**Pull request merged**